.github/workflows/autoupdate.yaml

```diff
@@ -0,0 +1,43 @@
+name: Generate Client
+on:
+ ## For verification only
+ push:
+   branches:
+     - CLOUDP-155908
```

Comment:
Do you mean to commit the CLOUDP as part of the automation? Seems the branch should be v2 or similar. If this is a prerelease branch, maybe use that name instead?

I'd expect after this commit, you can use the automation to publish an SDK, even if it's a private branch.

Reply:
```suggestion
 push:
   branches:
     - v2
```

Reply:
Made change to verify automation. Not needed anymore.